src/components/Content.js

```diff
@@ -14,7 +15,11 @@ const Content = ({ props, state, methods }) => {
       }`}
       onClick={(event) => {
```

Comment:
The click is on the "Content" component and the prop is called closeOnClick**Input**
What do you think? 
Is closeOnClick**Content** sounds legit I am not sure?

Reply:
Well yeah,
I thought for the usage it would be more accurate.

Why?
- What is the content, content can be just the multiselect chips or label, i mean so general.
- Calling it as Input, well the whole Select component we can say it is Input right.

But of course ur call

Reply:
Yep sounds legit.